guard/src/rules/eval_context.rs

```diff
@@ -2290,7 +2350,7 @@ pub(crate) fn resolve_function<'value, 'eval, 'loc: 'value>(
                 Ok(args)
             })?;
 
-    Ok(try_handle_function_call(name, &args)?
+    Ok(try_handle_function_call(name.to_owned(), &args)?
```

Comment:
Nit: Any reason why we dont update the `try_handle_function_call` to use `&FunctionName` ? I dont think the owned value is necessary here.

Reply:
Good call - will update.